recipes/window_ops/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: window_ops
+  version: 0.0.12
+source:
+  sha256: dc56e64b361533e659b339a940405e8c6582aab31a822e12485db8239de82804
+  url: https://files.pythonhosted.org/packages/09/cf/90624db95c1797d6db0686318152d74d941a893d2f9a1da731e5f434e3fd/window_ops-0.0.12.tar.gz
+build:
+  noarch: python
+  number: '0'
+  script: '{{ PYTHON }} -m pip install . -vv'
+requirements:
+  host:
+  - pip
+  - python >=3.6
+  - packaging
+  - numba
+  - numpy
+  run:
+  - pip
+  - python >=3.6
+  - packaging
+  - numba
+  - numpy
+test:
+  imports:
+  - window_ops
+about:
+  dev_url: https://jose-moralez.github.io
```

Comment:
These URLs don't actually work (they're missing the `window_ops` part of the URL.

Reply:
Thank you! I've fixed them in c85ca92